runenvweekly2all.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+set -e
```

Comment:
I am not a bash pro - but would that mean also the actual notification script would abort on error? For example if an error is thrown, because a telegram group removed permissions for our bot?